tests/src/whisk/core/dispatcher/test/DispatcherTests.scala

```diff
@@ -93,34 +92,4 @@ class DispatcherTests extends FlatSpec with Matchers with BeforeAndAfter with Be
         println(logs)
         logs should include regex (s"received message for 'whisk'.+$today.+")
     }
-
-    it should "receive message from post" in {
-        val config = new WhiskConfig(whisk.core.dispatcher.Dispatcher.requiredProperties ++ PostInvoke.requiredProperties)
-        assert(config.isValid)
-
-        val today = Calendar.getInstance.getTime.toString
-        val msg = Message(TransactionId.testing, "", Subject(), ActivationId(), Some(JsObject("payload" -> JsString(today))))
-        val namespace = Namespace("post test namespace")
-        val dispatcher = new TestDispatcher("invoke0")
```

Comment:
The `TestDispatcher` is still used for the test above, which is not related to the activator.